### ISSUE TITLE
feature request allow the rezume epesode cue to be cleared when running gpo

### ISSUE THREAD

**Issue by mhussaincov93** [gpodder/gpodder#1392]
hi,
would it be possible to
have the ability to clear remaining podcasts that are to be downloaded in cue?
when running via gpo?
when running via gpodder-gtk, this is not required because you have the in progress tab.
note,
I am unable to run gpodder-gtk due to orca screen reader freezing too many podcasts?
Majid

**Comment by auouymous** [comment]
I don't even see a way to clear a single partial episode, let alone all of them. That should definitely be a feature.

Until implemented, you can delete all files matching `gPodder/Downloads/*/*.partial*`, where "gPodder" is the gpodder directory. On linux that would be done using `rm ~/gPodder/Downloads/*/*.partial*` and I believe Windows uses `del` instead of `rm`, but has a different path to the gPodder directory.

**Comment by mhussaincov93** [comment]
cheers!
is this how the gpodder-gtk univirse works?
deletes all .partial files?
thanks again

**Comment by auouymous** [comment]
No, gpodder-gtk allows partials to be deleted from the UI, something gpo needs so the user doesn't need to manually go delete those partial files.
